modules/vcf2maf/1.1/vcf2maf.smk

```diff
@@ -91,18 +92,50 @@ rule _vcf2maf_run:
         > {log.stdout} 2> {log.stderr}
         """)
 
+rule _vcf2maf_crossmap:
+    input:
+        maf = rules._vcf2maf_run.output.maf,
+        convert_coord = CFG["inputs"]["convert_coord"]
+    output:
+        maf =  CFG["dirs"]["crossmap"] + "{seq_type}--{genome_build}/{tumour_id}--{normal_id}--{pair_status}/{base_name}.converted.maf"
+    log:
+        stdout = CFG["logs"]["crossmap"] + "{seq_type}--{genome_build}/{tumour_id}--{normal_id}--{pair_status}/{base_name}_crossmap.stdout.log",
+        stderr = CFG["logs"]["crossmap"] + "{seq_type}--{genome_build}/{tumour_id}--{normal_id}--{pair_status}/{base_name}_crossmap.stderr.log"
+    params:
+        chain = op.switch_on_wildcard("genome_build", CFG["switches"]["chains"])
+    conda:
+        CFG["conda_envs"]["crossmap"]
+    threads:
+        CFG["threads"]["vcf2maf"]
+    resources:
+        mem_mb = CFG["mem_mb"]["vcf2maf"]
+    shell:
+        op.as_one_line("""
+        {input.convert_coord}
+        {input.maf}
+        {params.chain}
+        {output.maf}
+        crossmap
+        > {log.stdout} 2> {log.stderr}
+        """)
+
+
 rule _vcf2maf_output_maf:
     input:
-        maf = str(rules._vcf2maf_run.output.maf)
```

Comment:
Why did you get rid of str() on this?

Reply:
When I added another input for converted files, snakemake was behaving somehow strange on them if wrapped with `str()`. I removed it here to keep consistent with the other input for converted files. I will return the `str()` wrapping here back. Since outputs of this rule were already available when I did the test, I did not run this rule explicitly, so it might not behave as I am thinking it would be.

Reply:
I can't remember why we started consistently adding str() but there was a good reason and it seemed to fix an issue I was having.